_posts/2019-03-09-rest-response-size.md

```diff
@@ -0,0 +1,138 @@
+---
+layout: post
+title:  "Myth: REST Responses Sizes are massive!"
+date:   2019-03-04 19:08:05 +0000
+categories: jekyll update
+---
+
+One of the biggest complaints people have about REST is that the response size is too big. In reality though, that is not a REST problem, that is a problem with the developers maintaining that endpoint. Let's look at some reasons why and how we can mitigate it. 
```

Comment:
Oh shit very first thing maybe you could say "This argument is a huge misnomer because OData and JSON-API, etc have sparse fieldsets (you can request only the fields you want" which is almost certainly the inspiration for the GraphQL feature.